plugins/invisible-typing/src/Settings.tsx

```diff
@@ -16,7 +16,7 @@ export function Settings(): React.ReactElement {
       </SwitchItem>
       <SwitchItem
         {...util.useSetting(cfg, "invisible", true)}
-        note="When enabled, invisibility applies to all channels. When disabled, invisibility is channel based.">
+        note="Whether invisibility applies to all channels. (Serves as default for channel's state when invisiblity is channel based)">
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Clarification in the settings UI about global invisibility.

This update in the description provides clearer guidance to the user on how the global invisibility setting interacts with channel-specific settings, enhancing usability.

```suggestion
note="Invisibility can be applied globally to all channels or individually per channel. When enabled globally, it sets the default state for new channels but can be overridden on a channel-by-channel basis.">
```